cmd/meroxa/root/connectors/describe.go

```diff
@@ -70,7 +70,7 @@ func (d *Describe) Execute(ctx context.Context) error {
 	return nil
```

Comment:
Not sure if this is part of the refactor, but you might wanna modify the Execute fn: https://github.com/meroxa/cli/pull/217/files#diff-36603a939a11613ba22e3770d59fc2369f352994353cd5838259f60272fe772dR61-R69

```
func (d *Describe) Execute(ctx context.Context) error {
	connector, err := d.client.GetConnectorByNameOrID(ctx, d.args.NameOrID)
	if err != nil {
		return err
	}
...
```

Reply:
the Get functions for all entity types are still divided into just id or just name. until we can change the pattern everywhere to one Get function that processes either name or ID, I just left that pattern as is.